api/src/pcapi/workers/apps_flyer_job.py

```diff
@@ -14,3 +17,27 @@ def log_user_becomes_beneficiary_event_job(user_id: int) -> None:
         log_user_event(user, f"af_complete_beneficiary_{user.age}")
     elif user.age >= 18:
         log_user_event(user, "af_complete_beneficiary_18")
+
+
+@job(worker.default_queue)
+def log_user_registration_event_job(user_id: int) -> None:
+    user = User.query.get(user_id)
+    if user.age == 15:
+        log_user_event(user, "af_complete_registration_15")
+    elif user.age == 16:
+        log_user_event(user, "af_complete_registration_16")
+    elif user.age == 17:
+        log_user_event(user, "af_complete_registration_17")
+    elif user.age == 18:
+        log_user_event(user, "af_complete_registration_18")
+    elif user.age >= 19:
+        log_user_event(user, "af_complete_registration_19+")
+
+
+@job(worker.default_queue)
+def log_user_booked_offer_event_job(user_id: int, offer_id: int, booking_id: int) -> None:
+    user = User.query.get(user_id)
+    offer = Offer.query.get(offer_id)
+    booking = Booking.query.get(booking_id)
+
+    log_offer_event(offer, booking, user, "af_complete_book_offer")
```

Comment:
On peut simplifier ici, en ne passant que le booking, qui a toutes les infos

user_id = booking.user.id
stock_id = booking.stock.id

```suggestion
@job(worker.default_queue)
def log_user_booked_offer_event_job(booking_id: int) -> None:
    booking = Booking.query.get(booking_id)
    # On peut joineldload les trucs utiles même
    booking = Booking.query.get(booking_id).joinedload(Booking.user).joinedload(Booking.stock).joinedload(Stock.offer)

    log_offer_event(booking, "af_complete_book_offer")
```

Puis dans log_offer_event
```python
def log_offer_event(booking: Booking, event_name: str) -> None:
    [...]
    data = {
        "appsflyer_id": context.apps_flyer_user_id,
        "eventName": event_name,
        "eventValue": {
            "af_user_id": str(booking.userId),
            "af_offer_id": booking.stock.offerId,
            "af_booking_id": booking.id,
            "af_price": booking.amount,
            "af_category": booking.stock.offer.subcategoryId,
        },
    }
```

Reply:
Done ✅